aten/src/ATen/native/native_functions.yaml

```diff
@@ -204,6 +204,10 @@
 
 - func: as_strided(Tensor(a) self, int[] size, int[] stride, int? storage_offset=None) -> Tensor(a)
   variants: function, method
+  dispatch:
```

Comment:
looks like adding dispatch here broke dispatch for bool tensors (our codegen is crazy). Probably the fix is to add
```
  cpu_half: True
  cpu_bool: True
  cuda_bool: True
```
here. @gchanan @smessmer to double check